perf/roles/trex/defaults/main.yml

```diff
@@ -2,5 +2,8 @@
 
 trex_repo: https://github.com/rh-nfv-int/trex-core
 trex_tag: rhosp-playbooks
-trex_dir: /tmp/trex-core
 trex_opts: ''
+trex_release_url: "https://trex-tgn.cisco.com/trex/release/latest"
+trex_release_filename: "trex_latest.tar.gz"
```

Comment:
what is this for?

Reply:
These are not required now, I removed it ... Earlier I am downloading the tar file first, so I kept a name variable for that. But, it is not required now.